src/web/lib/components/App/index.js

```diff
@@ -6,8 +6,10 @@ import { actions, selectors } from '../../../../lib/store';
 import AppBackground from '../AppBackground';
 import BrowserPreview from '../BrowserPreview';
 import ThemeColorsEditor from '../ThemeColorsEditor';
+import PresetThemeSelector from '../PresetThemeSelector';
 import ThemeBackgroundPicker from '../ThemeBackgroundPicker';
 import ExtensionInstallButton from '../ExtensionInstallButton';
+import ThemeUrl from '../ThemeUrl';
```

Comment:
To make the BrowserPreview component more composable, i moved the in content display of the url into it's own component. Right now, this component is stubbed out and does not accept any props.